src/main/java/com/feywild/feywild/entity/goals/AddShieldGoal.java

```diff
@@ -37,17 +37,17 @@ public void tick() {
             if (this.ticksLeft <= 0) {
                 this.addShieldEffect();
                 this.reset();
-            } else if (this.ticksLeft == 110) {
+            } else if (this.ticksLeft == 65) {
```

Comment:
Out of interest: Have you figured out what happened to the ticks here or did you just adjust them to some new values that work?

Reply:
I don't know what changed with the tick. 
So I just adjusted them all.